internal/editor/tempfile.go

```diff
@@ -0,0 +1,34 @@
+package editor
+
+import (
+	"fmt"
+	"os"
+)
+
+func temporaryFileNamePattern(marshalMode MarshalMode) string {
+	pattern := "scw-updateResourceEditor"
+	switch marshalMode {
+	case MarshalModeYAML:
+		pattern += "*.yml"
+	case MarshalModeJSON:
+		pattern += "*.json"
+	}
+	return pattern
+}
+
+func createTemporaryFile(content []byte, marshalMode MarshalMode) (string, error) {
```

Comment:
Would it be possible to have a default template that contains for each command a little template. Right now let's suppose that I do a:

```
scw instance security-group create
```

I get an empty security group and if I want to edit it, the temp file looks like:

```
rules: null
```

As a user, it is not very easy to guess what the final file should look like, I have no completion with the schema for instance. What could be useful is to have a little comment in this file that give an example of what the file should look like.